compiler/src/dotty/tools/dotc/interactive/Interactive.scala

```diff
@@ -105,6 +105,7 @@ object Interactive {
           buf ++= prefix.member(name).altsWith{ d =>
             !d.isAbsent &&
             !d.is(Synthetic) && !d.is(Artifact) &&
+            !d.is(ModuleClass) &&
```

Comment:
I think there's a better fix possible. Notice that we use `completionsFilter` to restrict the symbols we complete on, the definition below is:
```scala
     def apply(pre: Type, name: Name)(implicit ctx: Context): Boolean =
       !name.isConstructorName && name.is(SimpleNameKind)
```
This is supposed to filter out all the non-simple names but the check is wrong `name.is(SimpleNameKind)` will also return true if the underlying name kind is `SimpleNameKind`, the check should be replaced by ` name.toTermName.info.kind == SimpleNameKind`. If you do this then you won't have to explicitly filter out `ModuleClass`.

Reply:
Thank you for explanation!